Formula/git-webui.rb

```diff
@@ -0,0 +1,22 @@
+class GitWebui < Formula
+  desc "Local web based user interface for git repositories"
+  homepage "https://github.com/alberthier/git-webui"
+  url "https://github.com/alberthier/git-webui/archive/v1.2.0.tar.gz"
+  sha256 "21faa8a018d7325bd3acb7e7da138a2a61b504698f96fd067fa0ee765f3f15dd"
+  head "https://github.com/alberthier/git-webui.git"
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Should be an empty line between the end of the initial block & the first `depends_on`.